ompi/mca/osc/rdma/osc_rdma_passive_target.c

```diff
@@ -263,6 +339,63 @@ int ompi_osc_rdma_lock_atomic (int lock_type, int target, int assert, ompi_win_t
     return ret;
 }
 
+int ompi_osc_rdma_ilock_atomic (int lock_type, int target, int assert, ompi_win_t *win)
```

Comment:
I think would be better called `trylock` rather than `ilock` because it attempts to take a lock but does not return a request that can be waited on or tested until completion of the locking operation. If it fails to obtain the lock, it just returns a non-success value - the calling code must call this function again (rather than calling some sort of wait or test function).

Reply:
ilock->trylock